infra/build_deploy_config.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+
+import os
+import boto3
+import argparse
+
+
+infra_root = os.path.abspath(os.path.dirname(__file__))
```

Comment:
I think global variables are usually fully capitalized? Please check linter. Ditto in a few other locations.

Reply:
All caps is usually reserved for constants.